### PR TITLE
Supersense deprecation

### LINKED ISSUE #154: Deprecation policy
We need a story for what happens when the guidelines change in a structural way (beyond altering descriptions or examples):

- If a supersense, construal, or usage is removed, do we keep the old page around with a deprecation message? Should the models have a "deprecated" field so deprecated items are excluded from lists/counts and links to them are displayed with a strikethrough?
  * Should there be a deprecation message field? A way to view the history of deprecations?
- When a new version of a corpus is imported, what happens? Should there be a way in the site to access old corpus examples? What about displaying old corpus examples  for a deprecated supersense, construal, or usage?
  * If we include old examples they should clearly be marked as deprecated.
  * Or, the site could ignore old versions of corpora entirely, though they'll be available in the database.

### PR DESCRIPTION
**Pull request by lgessler**
Fixes #154 for supersenses

### CODE REVIEW

**Comment by nschneid** [comment]
There are some more places to surface the deprecation. I deprecated "Whole" but the Agent page is showing it via the `ss` macro without a strikethrough. Also, construals using the supersense should probably indicate that supersense is deprecated (the Whole page has a bunch of examples): e.g. Time↝~Whole~

**Comment by nschneid** [comment]
Also the `exref` macro

File: src/wiki/plugins/macros/mdx/macro.py [hunk 1]
```diff
@@ -205,29 +205,37 @@ def pspecial(self, **kwargs):
 
     def ss(self, **kwargs):
         args = argize_kwargs(kwargs)
-        cl = None
+
+        construal = None
```
Comment (nschneid) [inline src/wiki/plugins/macros/mdx/macro.py]: `cl` here was short for "class", not "construal". It's the "optional class" arg of the macro.

Comment (nschneid) [reply src/wiki/plugins/macros/mdx/macro.py]: `cls` is also used below (I don't recall why we needed separate variables)

File: src/wiki/plugins/macros/mdx/macro.py [hunk 2]
```diff
@@ -381,3 +389,9 @@ def argize_kwargs(kwargs):
         args.append(kwargs[arg])
     return args
 
+def strikethrough_if_deprecated(entry, elt):
```
Comment (nschneid) [inline src/wiki/plugins/macros/mdx/macro.py]: I'd call this `show_as_deprecated` (strikethrough per se is a CSS detail)

**Comment by lgessler** [comment]
@nschneid have another look at the dev server when you can, I think we've covered everything now

**Comment by lgessler** [comment]
Oops forget exref. Doing that now

**Pull request merged**